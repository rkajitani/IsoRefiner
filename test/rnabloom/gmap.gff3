##gff-version   3
# Generated by GMAP version 2024-11-20 using call:  gmap.sse42 -D . -d genome_gmap -t 4 -f 2 -n 1 --no-chimeras --min-trimmed-coverage=0.5 --min-identity=0.95 --max-intronlength-middle=100000 ./rnabloom_out/rnabloom.transcripts.fa
chr22	genome_gmap	gene	50257562	50261170	.	-	.	ID=rb_3.path1;Name=rb_3;Dir=antisense
chr22	genome_gmap	mRNA	50257562	50261170	.	-	.	ID=rb_3.mrna1;Name=rb_3;Parent=rb_3.path1;Dir=antisense;coverage=97.8;identity=100.0;matches=500;mismatches=0;indels=0;unknowns=0
chr22	genome_gmap	exon	50260829	50261170	100	-	.	ID=rb_3.mrna1.exon1;Name=rb_3;Parent=rb_3.mrna1;Target=rb_3 162 503 -
chr22	genome_gmap	exon	50258243	50258301	100	-	.	ID=rb_3.mrna1.exon2;Name=rb_3;Parent=rb_3.mrna1;Target=rb_3 103 161 -
chr22	genome_gmap	exon	50257562	50257660	100	-	.	ID=rb_3.mrna1.exon3;Name=rb_3;Parent=rb_3.mrna1;Target=rb_3 4 102 -
chr22	genome_gmap	CDS	50260829	50261170	100	-	0	ID=rb_3.mrna1.cds1;Name=rb_3;Parent=rb_3.mrna1;Target=rb_3 162 503 .
chr22	genome_gmap	CDS	50258243	50258301	100	-	0	ID=rb_3.mrna1.cds2;Name=rb_3;Parent=rb_3.mrna1;Target=rb_3 103 161 .
chr22	genome_gmap	CDS	50257648	50257660	100	-	2	ID=rb_3.mrna1.cds3;Name=rb_3;Parent=rb_3.mrna1;Target=rb_3 90 102 .
###
chr22	genome_gmap	gene	38687926	38689323	.	-	.	ID=rb_2.path1;Name=rb_2;Dir=antisense
chr22	genome_gmap	mRNA	38687926	38689323	.	-	.	ID=rb_2.mrna1;Name=rb_2;Parent=rb_2.path1;Dir=antisense;coverage=98.9;identity=99.4;matches=462;mismatches=3;indels=0;unknowns=0
chr22	genome_gmap	exon	38688935	38689323	100	-	.	ID=rb_2.mrna1.exon1;Name=rb_2;Parent=rb_2.mrna1;Target=rb_2 77 465 -
chr22	genome_gmap	exon	38687926	38688001	96	-	.	ID=rb_2.mrna1.exon2;Name=rb_2;Parent=rb_2.mrna1;Target=rb_2 1 76 -
chr22	genome_gmap	CDS	38688935	38689098	100	-	0	ID=rb_2.mrna1.cds1;Name=rb_2;Parent=rb_2.mrna1;Target=rb_2 77 240 .
chr22	genome_gmap	CDS	38687926	38688001	96	-	2	ID=rb_2.mrna1.cds2;Name=rb_2;Parent=rb_2.mrna1;Target=rb_2 1 76 .
###
chr22	genome_gmap	gene	50252907	50261708	.	-	.	ID=rb_4.path1;Name=rb_4;Dir=antisense
chr22	genome_gmap	mRNA	50252907	50261708	.	-	.	ID=rb_4.mrna1;Name=rb_4;Parent=rb_4.path1;Dir=antisense;coverage=99.9;identity=100.0;matches=1767;mismatches=0;indels=0;unknowns=0
chr22	genome_gmap	exon	50261385	50261708	100	-	.	ID=rb_4.mrna1.exon1;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1444 1767 -
chr22	genome_gmap	exon	50261167	50261296	100	-	.	ID=rb_4.mrna1.exon2;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1314 1443 -
chr22	genome_gmap	exon	50258243	50258301	100	-	.	ID=rb_4.mrna1.exon3;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1255 1313 -
chr22	genome_gmap	exon	50257082	50257193	100	-	.	ID=rb_4.mrna1.exon4;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1143 1254 -
chr22	genome_gmap	exon	50256599	50256646	100	-	.	ID=rb_4.mrna1.exon5;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1095 1142 -
chr22	genome_gmap	exon	50256085	50256199	100	-	.	ID=rb_4.mrna1.exon6;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 980 1094 -
chr22	genome_gmap	exon	50255810	50255881	100	-	.	ID=rb_4.mrna1.exon7;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 908 979 -
chr22	genome_gmap	exon	50255615	50255694	100	-	.	ID=rb_4.mrna1.exon8;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 828 907 -
chr22	genome_gmap	exon	50255453	50255531	100	-	.	ID=rb_4.mrna1.exon9;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 749 827 -
chr22	genome_gmap	exon	50255197	50255370	100	-	.	ID=rb_4.mrna1.exon10;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 575 748 -
chr22	genome_gmap	exon	50252907	50253480	100	-	.	ID=rb_4.mrna1.exon11;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1 574 -
chr22	genome_gmap	CDS	50261385	50261707	100	-	0	ID=rb_4.mrna1.cds1;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1444 1766 .
chr22	genome_gmap	CDS	50261167	50261296	100	-	2	ID=rb_4.mrna1.cds2;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1314 1443 .
chr22	genome_gmap	CDS	50258243	50258301	100	-	0	ID=rb_4.mrna1.cds3;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1255 1313 .
chr22	genome_gmap	CDS	50257082	50257193	100	-	2	ID=rb_4.mrna1.cds4;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1143 1254 .
chr22	genome_gmap	CDS	50256599	50256646	100	-	0	ID=rb_4.mrna1.cds5;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 1095 1142 .
chr22	genome_gmap	CDS	50256085	50256199	100	-	0	ID=rb_4.mrna1.cds6;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 980 1094 .
chr22	genome_gmap	CDS	50255810	50255881	100	-	1	ID=rb_4.mrna1.cds7;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 908 979 .
chr22	genome_gmap	CDS	50255615	50255694	100	-	1	ID=rb_4.mrna1.cds8;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 828 907 .
chr22	genome_gmap	CDS	50255453	50255531	100	-	0	ID=rb_4.mrna1.cds9;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 749 827 .
chr22	genome_gmap	CDS	50255197	50255370	100	-	1	ID=rb_4.mrna1.cds10;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 575 748 .
chr22	genome_gmap	CDS	50253401	50253480	100	-	1	ID=rb_4.mrna1.cds11;Name=rb_4;Parent=rb_4.mrna1;Target=rb_4 495 574 .
###
chr22	genome_gmap	gene	38700277	38701319	.	-	.	ID=rb_6.path1;Name=rb_6;Dir=sense
chr22	genome_gmap	mRNA	38700277	38701319	.	-	.	ID=rb_6.mrna1;Name=rb_6;Parent=rb_6.path1;Dir=sense;coverage=98.3;identity=96.8;matches=399;mismatches=1;indels=12;unknowns=0
chr22	genome_gmap	exon	38701257	38701319	100	-	.	ID=rb_6.mrna1.exon1;Name=rb_6;Parent=rb_6.mrna1;Target=rb_6 8 70 +
chr22	genome_gmap	exon	38700277	38700618	96	-	.	ID=rb_6.mrna1.exon2;Name=rb_6;Parent=rb_6.mrna1;Target=rb_6 71 414 +
chr22	genome_gmap	CDS	38701257	38701319	100	-	0	ID=rb_6.mrna1.cds1;Name=rb_6;Parent=rb_6.mrna1;Target=rb_6 8 70 +
chr22	genome_gmap	CDS	38700397	38700618	95	-	0	ID=rb_6.mrna1.cds2;Name=rb_6;Parent=rb_6.mrna1;Target=rb_6 71 292 +
###
chr22	genome_gmap	gene	38685579	38700571	.	-	.	ID=rb_5.path1;Name=rb_5;Dir=sense
chr22	genome_gmap	mRNA	38685579	38700571	.	-	.	ID=rb_5.mrna1;Name=rb_5;Parent=rb_5.path1;Dir=sense;coverage=100.0;identity=99.9;matches=3515;mismatches=0;indels=3;unknowns=0
chr22	genome_gmap	exon	38699803	38700571	100	-	.	ID=rb_5.mrna1.exon1;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 2 770 +
chr22	genome_gmap	exon	38689296	38689424	100	-	.	ID=rb_5.mrna1.exon2;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 771 899 +
chr22	genome_gmap	exon	38688935	38689129	100	-	.	ID=rb_5.mrna1.exon3;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 900 1094 +
chr22	genome_gmap	exon	38685579	38688001	99	-	.	ID=rb_5.mrna1.exon4;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 1095 3518 +
chr22	genome_gmap	CDS	38699803	38699987	100	-	0	ID=rb_5.mrna1.cds1;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 586 770 +
chr22	genome_gmap	CDS	38689296	38689424	100	-	2	ID=rb_5.mrna1.cds2;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 771 899 +
chr22	genome_gmap	CDS	38688935	38689129	100	-	2	ID=rb_5.mrna1.cds3;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 900 1094 +
chr22	genome_gmap	CDS	38687934	38688001	98	-	2	ID=rb_5.mrna1.cds4;Name=rb_5;Parent=rb_5.mrna1;Target=rb_5 1095 1161 +
###
