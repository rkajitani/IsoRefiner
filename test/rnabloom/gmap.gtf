chr22	genome_gmap	transcript	38685579	38700571	.	-	.	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1"
chr22	genome_gmap	exon	38685579	38688001	99	-	.	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	exon	38688935	38689129	100	-	.	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	exon	38689296	38689424	100	-	.	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	exon	38699803	38700571	100	-	.	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	CDS	38687934	38688001	.	-	1	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	CDS	38688935	38689129	.	-	1	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	CDS	38689296	38689424	.	-	1	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	CDS	38699803	38699987	.	-	0	transcript_id "rb_5.mrna1"; gene_id "rb_5.path1";
chr22	genome_gmap	transcript	38687926	38689323	.	-	.	transcript_id "rb_2.mrna1"; gene_id "rb_2.path1"
chr22	genome_gmap	exon	38687926	38688001	96	-	.	transcript_id "rb_2.mrna1"; gene_id "rb_2.path1";
chr22	genome_gmap	exon	38688935	38689323	100	-	.	transcript_id "rb_2.mrna1"; gene_id "rb_2.path1";
chr22	genome_gmap	CDS	38687926	38688001	.	-	1	transcript_id "rb_2.mrna1"; gene_id "rb_2.path1";
chr22	genome_gmap	CDS	38688935	38689098	.	-	0	transcript_id "rb_2.mrna1"; gene_id "rb_2.path1";
chr22	genome_gmap	transcript	38700277	38701319	.	-	.	transcript_id "rb_6.mrna1"; gene_id "rb_6.path1"
chr22	genome_gmap	exon	38700277	38700618	96	-	.	transcript_id "rb_6.mrna1"; gene_id "rb_6.path1";
chr22	genome_gmap	exon	38701257	38701319	100	-	.	transcript_id "rb_6.mrna1"; gene_id "rb_6.path1";
chr22	genome_gmap	CDS	38700397	38700618	.	-	0	transcript_id "rb_6.mrna1"; gene_id "rb_6.path1";
chr22	genome_gmap	CDS	38701257	38701319	.	-	0	transcript_id "rb_6.mrna1"; gene_id "rb_6.path1";
chr22	genome_gmap	transcript	50252907	50261708	.	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1"
chr22	genome_gmap	exon	50252907	50253480	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50255197	50255370	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50255453	50255531	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50255615	50255694	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50255810	50255881	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50256085	50256199	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50256599	50256646	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50257082	50257193	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50258243	50258301	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50261167	50261296	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	exon	50261385	50261708	100	-	.	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50253401	50253480	.	-	2	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50255197	50255370	.	-	2	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50255453	50255531	.	-	0	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50255615	50255694	.	-	2	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50255810	50255881	.	-	2	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50256085	50256199	.	-	0	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50256599	50256646	.	-	0	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50257082	50257193	.	-	1	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50258243	50258301	.	-	0	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50261167	50261296	.	-	1	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	CDS	50261385	50261707	.	-	0	transcript_id "rb_4.mrna1"; gene_id "rb_4.path1";
chr22	genome_gmap	transcript	50257562	50261170	.	-	.	transcript_id "rb_3.mrna1"; gene_id "rb_3.path1"
chr22	genome_gmap	exon	50257562	50257660	100	-	.	transcript_id "rb_3.mrna1"; gene_id "rb_3.path1";
chr22	genome_gmap	exon	50258243	50258301	100	-	.	transcript_id "rb_3.mrna1"; gene_id "rb_3.path1";
chr22	genome_gmap	exon	50260829	50261170	100	-	.	transcript_id "rb_3.mrna1"; gene_id "rb_3.path1";
chr22	genome_gmap	CDS	50257648	50257660	.	-	1	transcript_id "rb_3.mrna1"; gene_id "rb_3.path1";
chr22	genome_gmap	CDS	50258243	50258301	.	-	0	transcript_id "rb_3.mrna1"; gene_id "rb_3.path1";
chr22	genome_gmap	CDS	50260829	50261170	.	-	0	transcript_id "rb_3.mrna1"; gene_id "rb_3.path1";
