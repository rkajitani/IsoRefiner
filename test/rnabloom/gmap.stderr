Note: gmap.avx512 does not exist.  For faster speed, may want to compile package on an AVX512 machine
Note: gmap.avx2 does not exist.  For faster speed, may want to compile package on an AVX2 machine
GMAP version 2024-11-20 called with args: gmap.sse42 -D . -d genome_gmap -t 4 -f 2 -n 1 --no-chimeras --min-trimmed-coverage=0.5 --min-identity=0.95 --max-intronlength-middle=100000 ./rnabloom_out/rnabloom.transcripts.fa
Note: -n 1 will not report chimeric alignments.  If you want a single alignment plus chimeras, use -n 0 instead.
Checking compiler assumptions for SSE2: 6B8B4567 327B23C6 xor=59F066A1
Checking compiler assumptions for SSE4.1: -103 -58 max=198 => compiler zero extends
Checking compiler options for SSE4.2: 6B8B4567 __builtin_clz=1 __builtin_ctz=0 _mm_popcnt_u32=17 __builtin_popcount=17 
Finished checking compiler assumptions
Pre-loading compressed genome (oligos)......done (19,056,944 bytes, 4653 pages, 0.00 sec)
Looking for genome genome_gmap in directory ./genome_gmap
Looking for index files in directory ./genome_gmap
  Pointers file is genome_gmap.ref153offsets64meta
  Offsets file is genome_gmap.ref153offsets64strm
  Positions file is genome_gmap.ref153positions
Offsets compression type: bitpack64
Allocating memory for ref offset pointers, kmer 15, interval 3...done (134,217,744 bytes, 0.08 sec)
Allocating memory for ref offsets, kmer 15, interval 3...done (94,189,504 bytes, 0.06 sec)
Pre-loading ref positions, kmer 15, interval 3......done (52,212,140 bytes, 0.00 sec)
Starting alignment
No paths found for rb_1
Processed 6 queries in 0.09 seconds (68.34 queries/sec)
0.16user 0.20system 0:00.26elapsed 138%CPU (0avgtext+0avgdata 292868maxresident)k
0inputs+24outputs (0major+82818minor)pagefaults 0swaps
